Opta-Stats/OptaStats.md

```diff
@@ -0,0 +1,82 @@
+
+- `<app_schema_id>` **(Required)**: Application scheme id.
+- `<screen_id>` **(Required)**: A predefined `<screen_id>` from one of the following options:
+-- `home_screen`
+-- `match_details_screen`
+-- `all_matches_screen`
+-- `all_teams_screen`
+-- `team_screen`
+-- `player_screen`
+- `<other_data_key>` **(Required for specific screens)**: To show values from a specific entity:
+-- `match_id=<op_id>&push=<true_or_false>`
+-- `team_id` (`team_screen` and `all_matches_screen`)
+-- `player_id` (`player_screen`)
+
+#### Match details & Push option
+When opening match details screen through URL scheme, there is another key named push where indicates if the URL scheme comes from the push or if it’s being opened through the regular way.
+
+When the url scheme is coming from push the `match_id` is different from the one provided by Opta. Let's call this data, `match_id_from_push`. With this id provided by the push is necessary to send to Opta API as `match_id`, `urn:perform:opta:fixture:<match_id_from_push>`. The id coming from the push may have a letter at the beginning. It is necessary to remove this letter to use it as `match_id` in the API call.
+
+## Available screens
+The list of screen available in the integration will be the following:
+- [Home screen](#home-screen)
+- [Match Details Screen](#match-details-screen)
+- [All Matches Screen](#all-matches-screen)
+- [Team Screen](#team-screen)
+- [Player Screen](#player-screen)
+
+### <a name=home-screen></a> Home screen
+You can find the whole design [here](https://app.zeplin.io/project/5c8863d4039b9f39096d453d/screen/5c8864178bd661197ccb8c82).
+
+![Home screen preview](media/home_screen.png)
+
+### <a name=match-details-screen></a> Match Details Screen
+You can find the whole design [here](https://app.zeplin.io/project/5c8863d4039b9f39096d453d/screen/5c88641af85b69045c2eafca).
+
+![Match details screen preview](media/match_details_1.png) ![Match details screen preview](media/match_details_2.png) ![Match details screen preview](media/match_details_3.png)
+
+### <a name=all-matches-screen></a> All Matches Screen
+You can find the whole design [here](https://app.zeplin.io/project/5c8863d4039b9f39096d453d/screen/5c886418988ef419824d583b).
+
+![All matches screen preview](media/all_matches_screen.png)
+
+### <a name=team-screen></a> Team Screen
+You can find the whole design [here](https://app.zeplin.io/project/5c8863d4039b9f39096d453d/screen/5caf607f225f00a04b00e1d2).
+
+![Team screen preview](media/team_screen.png)
+
+### <a name=player-screen></a> Player Screen
+You can find the whole design [here](https://app.zeplin.io/project/5c8863d4039b9f39096d453d/screen/5cab7ce9e89d3f338980bcb3).
+
+![Player screen preview](media/player_screen.png)
+
+## Plugin configuration
+In order to make this plugin general and that can be used for other tournaments, we define the following parameters:
```

Comment:
In order to make this plugin ~general~ generic and ~that can~ be used for other tournaments, we define**d** the following parameters: